src/cdk/a11y/key-manager/tree-key-manager.ts

```diff
@@ -411,7 +411,11 @@ export class TreeKeyManager<T extends TreeKeyManagerItem> implements TreeKeyMana
   }
 }
 
-/** @docs-private */
+/**
+ * @docs-private
+ * @deprecated No longer used, will be removed.
+ * @breaking-change 21.0.0
```

Comment:
Shouldn't this be v22, since it'd be deprecated during two major releases (v20, v21)?

Reply:
The way I interpreted it is that it would have to be deprecated during two major versions: 19 and 20.